config_operator/config_operator/config_source/RemoteGitConfig.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+import sys
+import time
+from git import Repo
+from .LocalDirConfig import LocalDirConfig
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+LISTEN_FOR_UPDATE_INTERVAL_SECONDS = 5
```

Comment:
Same as my comment on line 12 of `LocalDirConfig.py`

Reply:
ok, done